easybuild/easyconfigs/t/tensorstore/tensorstore-0.1.72-gfbf-2024a.eb

```
@@ -373,6 +373,10 @@ _preinstall_bazel_exports_buildopts = 'export TENSORSTORE_BAZEL_BUILD_OPTIONS="'
 _preinstall_bazel_exports_buildopts += '$(for i in  $CFLAGS;do echo --copt=$i; done) '  # pass CFLAGS to Bazel as copts
 _preinstall_bazel_exports_buildopts += '%s %s' % (_bzl_linkopts, _bzl_include_copts)
 _preinstall_bazel_exports_buildopts += ' --distdir=%s' % _dist_dir
+# pass $TMPDIR from host into sandbox used by Bazel,
+# required when using RPATH linking because of RPATH wrapper scripts;
+# see also https://github.com/bazelbuild/bazel/issues/19915
+_preinstall_bazel_exports_buildopts += ' --sandbox_add_mount_pair=$TMPDIR'
```

```suggestion
_preinstall_bazel_exports_buildopts += ' --sandbox_add_mount_pair=$TMPDIR'
_preinstall_bazel_exports_buildopts += ' --subcommands --verbose_failures'
```

to further debug this issue